### PR TITLE
Link all components to panel UI

### LINKED ISSUE #138: Update User Interface to fit Homerce
Homerce Ui needs multiple tabs for each feature i.e appointments, services, etc. Let's add these components into the JavaFX GUI so that the application is more cohesive as a whole.

### PR DESCRIPTION
**Pull request by RuiFengg**
Link all the components to the UI panels

### CODE REVIEW

File: src/test/java/seedu/address/storage/client/JsonClientManagerStorageTest.java [hunk 1]
```diff
@@ -48,16 +48,19 @@ public void read_missingFile_emptyResult() throws Exception {
     @Test
     public void read_notJsonFormat_exceptionThrown() {
         assertThrows(DataConversionException.class, () -> readAddressBook("notJsonFormatClient.json"));
+        assertThrows(DataConversionException.class, () -> readAddressBook("notJsonFormatClient.json"));
```
Comment (khoongwk) [inline src/test/java/seedu/address/storage/client/JsonClientManagerStorageTest.java]: Do we need 2 lines of the same assertion?

Comment (khoongwk) [reply src/test/java/seedu/address/storage/client/JsonClientManagerStorageTest.java]: Lines 57, 63 and 111 as well

**Review by khoongwk — COMMENTED** [review]
Not sure why there are some duplicate assertion tests but other than that, seems good. Can merge after checkstyle issues are fixed.

File: src/test/java/seedu/address/storage/client/JsonClientManagerStorageTest.java (hunk 1 above)
Comment (RuiFengg) [reply src/test/java/seedu/address/storage/client/JsonClientManagerStorageTest.java]: Thx for the catch! Removed them :smile:

**Review by khoongwk — APPROVED** [review]
LGTM

**Pull request merged**